### PR TITLE
Slack notification on deployment

### PR DESCRIPTION
**Pull request by amalfra**
Create new slack notification when deployed.  #24

### CODE REVIEW

File: notifications/slack.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# Post to Slack channel on new deployment
+# https://api.slack.com/incoming-webhooks
+#
+# You can either add those here, or configure them on the environment tab of your
+# project settings.
+SLACK_WEBHOOK_TOKEN="T02CLSA7U/B051FFP0B/jtRHTrx7rp2un5rS3gsmZvKQ"
```
Comment (mlocher) [inline notifications/slack.sh]: You very likely want to **immediately** reset this webhook integration, or anybody can send messages to your channel!

Please never include tokens or other methods to authenticate with ressources in the PRs!

**Comment by mlocher** [comment]
What's the advantage of using this script and not the integrated Slack notifications? 

It would be great if you could make the message configurable via a parameter to the script and check the environment variables as done with e.g. https://github.com/codeship/scripts/blob/master/packages/sauce_connect.sh

``` bash
# default value when not specified
SAUCE_VERSION=${SAUCE_VERSION:="4.3.8"}

# error when not specified
SAUCE_USER=${SAUCE_USER:?'You need to configure the SAUCE_USER environment variable!'}
```

Ideally you'd configure the three environment variables (`SLACK_WEBHOOK_TOKEN`, `SLACK_BOT_NAME`, `SLACK_ICON_URL`) and then call the script as follows

``` bash
slack_message "${CI_COMMITTER_USERNAME} just deployed version ${CI_COMMIT_ID}"
slack_message "Running a new build on Codeship"
```

File: notifications/slack.sh (hunk 1 above)
Comment (amalfra) [inline notifications/slack.sh]: Ouch! My bad... I forgot to remove that. Anyway i had deleted the integration :)

**Comment by amalfra** [comment]
@mlocher Sure. Would do the changes.

**Comment by mlocher** [comment]
First, thanks so much for making the changes already. (And for deleting the token :) )

I'm afraid there was a misunderstanding regarding the purpose though. The script shouldn't make both calls, but allow the user the specify the message on the CLI as a parameter to the call to `slack_message`.

That is, the use should be able to specify any message he likes to have sent to Slack and the script should just send it. You don't need to care about variable expansion, as the shell already takes care of that, you only need to make sure there is a message specified and then send this message.

**Comment by amalfra** [comment]
@mlocher That should be fine. Would be pushing the changes in a while.

**Comment by mlocher** [comment]
Merci :)

**Comment by amalfra** [comment]
Thanks for merging :)

**Pull request merged**